spec/models/student_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+require 'date'
+
+RSpec.describe Student, :type => :model do
+
+  context "with the title 'Miss'" do
+    subject { FactoryBot.build(:student, title: "Miss")}
+    it { is_expected.to have_attributes(:title => "Miss") }
+  end
+
+  context "with the first_name 'Alice'" do
+    subject { FactoryBot.build(:student, first_name: "Alice")}
+    it { is_expected.to have_attributes(:first_name => "Alice") }
```

Comment:
Can you replace your hash syntax with direct symbols, please?

```suggestion
    it { is_expected.to have_attributes(first_name: "Alice") }
```